devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -378,18 +378,11 @@ def refresh() {
 }
 
 def configure() {
-	def requests = []
-
+	unschedule(scheduledUpdateWeather)
 	if (settings.zipcode) {
-		requests += scheduledUpdateWeather()
-		unschedule(scheduledUpdateWeather)
 		runEvery1Hour(scheduledUpdateWeather)
-	} else {
-		unschedule(scheduledUpdateWeather)
 	}
-	requests += poll()
-
-	requests
+	poll()
```

Comment:
poll() actually has a call to updateWeather, so no need for the unnecessary call to the same